planemo/runnable.py

```diff
@@ -225,7 +236,7 @@ def for_path(path: str, temp_path: Optional[str] = None, return_all: bool = Fals
     """Produce a class:`Runnable` for supplied path."""
     runnable_type = None
     if os.path.isdir(path):
-        runnable = workflow_dir_runnables(path, return_all=return_all)
+        runnable = workflow_dir_runnables(path, return_all=return_all) or tool_dir_runnables(path, temp_path)
         if runnable:
             return runnable
         runnable_type = RunnableType.directory
```

Comment:
Should we raise an exception if no runnable is found, or is there still a use case for continuing with the rest of the function?

Reply:
I think RunnableType.directory is a valid runnable, and we raise appropriate exceptions if a engine can't deal with a runnable. I think this is fine ?